frontend/packages/react-app/src/components/molecules/TokenBalance/index.tsx

```diff
@@ -1,20 +1,29 @@
 import React from "react";
 import { Box, Typography } from "@material-ui/core";
+import { getBalanceDevidedByDecimals } from "../../../utils/web3";
 
 export interface TokenInfo {
   readonly balance: string;
+  readonly decimals: number;
   readonly symbol: string;
   readonly itemName: string;
 }
 
-const TokenBalance: React.FC<TokenInfo> = ({ balance, symbol, itemName }) => (
+const TokenBalance: React.FC<TokenInfo> = ({
+  balance,
+  decimals,
+  symbol,
+  itemName,
+}) => (
   <>
     <Box display="flex" style={{ alignItems: "baseline" }}>
       <Box mr={4}>
         <Typography>{itemName}</Typography>
       </Box>
       <Box mr={1}>
-        <Typography variant={"h4"}>{balance}</Typography>
+        <Typography variant={"h4"}>
+          {getBalanceDevidedByDecimals(balance, decimals)}
+        </Typography>
```

Comment:
We have a `TokenAmount` component and you can use it. Is it not suitable? 
https://github.com/iroiro/iroiro/tree/develop/frontend/packages/react-app/src/components/atoms/TokenAmount

Reply:
Fixed